app/assets/stylesheets/application.css.scss

```diff
@@ -62,11 +62,11 @@ input {
   border: 0px;
 }
 
-.navbar-form > input[type="text"] {
-  width: 200px;
-  height: 30px;
-  border-radius: 4px;
-  border: 0px;
+.navbar-form > form > input[type="text"] {
+ border-radius: 4px;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered border, border-radius, height, width